### ISSUE TITLE
The `input` event dispatched in lieu of `tribute-replaced` is too general an event type

### ISSUE THREAD

**Issue by djramones** [syropian/vue-tribute#42]
This line I believe means that `vue-tribute` dispatches a (rather generic) `input` event so that the original `tribute-replaced` event can be communicated to the rest of the Vue app:

https://github.com/syropian/vue-tribute/blob/130604683b2520f134ae86722edbe5b02b63b598/src/index.js#L46

The problem with this is I want to just catch `tribute-replaced`, that is, to do something only when the user actually has selected a matched option. `input`, unfortunately, is also generated when the user is just typing for the keyword *before* selecting a match.

Trying to work around this, I changed the new event to also `tribute-replaced` instead of `input`, but that quickly produced some kind of infinite recursion. I ended up using a slightly different event name, `vue-tribute-replaced`, which has worked perfectly.

Incorporating this to the package would be a breaking change though for anyone who has used the wrapper and relied on the `input`-type event.

**Comment by djramones** [comment]
Two months later, I discovered that the ‘workaround’ I made that ‘worked perfectly’ actually broke something: there's a good reason `input` was chosen for that line of code, and that's to allow the Tribute event to propagate to Vue's internal view-model. That's kind of the point of this wrapper. (See: d46c0e0ee5fbb72f0e872f7a095711c82101b954) Changing the event to something else meant that the user-visible contents of the child/input element didn't synchronize with Vue's internal data after a `tribute-replaced` event, until after the user performed a further action that emitted an `input` event some other way (e.g., typed some more characters instead of submitting the form immediately, if Tribute is being used within a form).

In the end, I fixed my project's code by listening for Tribute-generated changes as recommended in this wrapper's readme, that is, by attaching an event listener for `tribute-replaced` directly on the child/input element, instead of some other way, messing with other event types.
